traffic-manager-application-gateway-demo-setup/azuredeploy.json

```diff
@@ -0,0 +1,222 @@
+      }
+    },
+    "secondarylocation": {
+      "type": "string",
+      "allowedValues": [
+        "East Asia",
+        "Southeast Asia",
+        "Central US",
+        "East US",
+        "East US 2",
+        "West US",
+        "North Central US",
+        "South Central US",
+        "North Europe",
+        "West Europe",
+        "Japan West",
+        "Japan East",
+        "Brazil South",
+        "Canada Central",
+        "Canada East"
+      ],
+      "defaultValue": "West US",
+      "metadata": {
+        "description": "Location of the secondary endpoint"
+      }
+    },
+    "trafficRoutingMethod": {
+      "type": "string",
+      "allowedValues": [
+        "Priority",
+        "Weighted",
+        "Performance"
+      ],
+      "defaultValue": "Weighted",
+      "metadata": {
+        "description": "Traffic routing methods available in Traffic Manager"
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "User name for the backend Web servers"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the backend Web servers"
+      }
+    }
+  },
+  "variables": {
+    "applicationGatewayName": "applicationGateway1",
+    "virtualNetworkName": "virtualNetwork1",
+    "subnetName": "appGatewaySubnet",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+    "applicationGatewayID": "[resourceId('Microsoft.Network/applicationGateways',variables('applicationGatewayName'))]",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetPrefix": "10.0.0.0/28",
+    "appGwTemplateLocation": "https://raw.githubusercontent.com/AlekseiPolkovnikov/azure-quickstart-templates/tmagw/traffic-manager-application-gateway-demo-setup/",
+    "appGwTemplateName": "azuredeployappgw.json"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-01-01",
+      "name": "appGw1",
+      "type": "Microsoft.Resources/deployments",
+      "dependsOn": [
+        "[concat('Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'))]"
+      ],
+      "properties": {
+        "mode": "incremental",
+        "templateLink": {
+          "uri": "[concat(variables('appGwTemplateLocation'), '/', variables('appGwTemplateName'))]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "adminUsername": {
+            "value": "[parameters('adminUsername')]"
+          },
+          "adminPassword": {
+            "value": "[parameters('adminPassword')]"
+          },
+          "location": {
+            "value": "[parameters('primarylocation')]"
+          },
+          "appGwName": {
+            "value": "appgw1"
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "2015-01-01",
+      "name": "appGw2",
```

Comment:
This was considered, however, having only 2 instances, input that is parameter names bound (parameters(primarylocation'))/parameters('secondarylocation')), and deployments output handling, looked much easier?